notifications.bs

```diff
@@ -729,8 +735,20 @@ constructor steps are:
 return the <a>permission</a> for the <a>current settings object</a>'s
 <a for="environment settings object">origin</a>.
 
-<p class=note>If you edit standards please refrain from copying the above. Synchronous
-permissions are like synchronous IO, a bad idea.
+<div class=note>
+ <p>If you edit standards please refrain from copying the above. Synchronous permissions are like
+ synchronous IO, a bad idea.
+
+ <p>Developers are encouraged to use the [[Permissions]] API's {{Permissions/query()}} method
```

Comment:
We typically write the name of the standard in full in the sentence and put the reference at the end of the paragraph, after the dot.